pkg/core/native/native_neo.go

```diff
@@ -43,16 +43,17 @@ type NeoCache struct {
 
 	registerPrice int64
 
+	validators keys.PublicKeys
```

Comment:
Why moving it?

Reply:
It didn't depend on boolean flag field, so I desided not to squash it with the group of flag-dependant fields.

Reverted.